src/model/interactions/mod.rs

```diff
@@ -64,6 +64,16 @@ impl Interaction {
         }
     }
 
+    /// Gets the invoked guild locale.
+    pub fn guild_locale(&self) -> Option<&str> {
+        match self {
+            Interaction::Ping(ref i) => i.guild_locale.as_ref().map(String::as_str),
```

Comment:
`ref` is implied when the expression in `match <expr>` is a reference.
```suggestion
            Interaction::Ping(i) => i.guild_locale.as_ref().map(String::as_str),
```